non-free.md

```diff
@@ -33,7 +33,7 @@
 
 ### File Transfer & Synchronization
 
-- [Dropcenter](http://projet.idleman.fr/dropcenter/) - Upload files by simple drag-n-drop. ([Source Code](https://github.com/ldleman/dropcenter)) `CC-BY-NC-SA-3.0` `PHP`
+- [Dropcenter](http://projet.idleman.fr/login) - Upload files by simple drag-n-drop. ([Source Code](https://github.com/ldleman/dropcenter)) `CC-BY-NC-SA-3.0` `PHP`
```

Comment:
> https://github.com/ldleman/dropcenter

no update since 2018

> le dépot a été migré sur le serveur suivant : https://git.idleman.fr/idleman/dropcenter

https://git.idleman.fr/idleman/dropcenter: error 404

> http://projet.idleman.fr/login

owncloud login page, unrelated.

Let's remove Dropcenter completely